CHANGELOG.md

```diff
@@ -24,6 +24,25 @@ forget to update the links at the bottom of the changelog as well.-->
 ### Documentation
 
 
+## [v0.9.0] - 2021-12-09
+
+### Additions
+
+- Unify RCC enable / reset ([#196])
```

Comment:
Again: Not sure if this is an addition or breaking / non-breaking change.

Reply:
Strictly speaking it's breaking, due to the addition of trait bounds to at least one `Instance` trait. I can't imagine how that could matter to a user.